src/modules/auth/Auth.page.tsx

```diff
@@ -0,0 +1,8 @@
+import useLazyLoadResourceHook from '@shared/hooks/lazyLoadResource.hook';
+import { SignUp } from './components/SignUp';
+
+export default function Auth(): JSX.Element {
+  useLazyLoadResourceHook({ folderName: 'Auth', namespace: 'auth' });
```

Comment:
Iz nekog razloga git nije povukao moje promene, kad sam preimenovao folder `home` u `Home`... Tako da svuda gde se koristi `useLazyLoadResourceHook` treba `folderName` da se izmeni tako da se poklapa sa imenom foldera u kom se nalaze fajlovi za lokalizaciju.
Npr. u `src/modules/home/Home.page.tsx` 
`useLazyLoadResourceHook({ folderName: "Home", namespace: "home" });` treba da predje u `useLazyLoadResourceHook({ folderName: "home", namespace: "home" });`

Mozda malo da ti bude jasnije, ovo `folderName: "home"` predstavlja ime modula.

E sad, i kad popravis ovaj nas (moji i Tosin) propust... Moji jer nisam primetio da preimenovani folderi nisu otisli kao promena u PR... A Tosina jer mi nije rekoa da fja baca error... 

Treba da dodas folder i fajlove za lokalizaciju.
To uradi po uglednu na `locales` folder is `home` modula.